sass/screen.scss

```diff
@@ -95,97 +129,159 @@ img {
   top: -1px
 }
 
-#lrg-mark {
-    background-image: url("../images/Logo.png");
-    background-repeat: no-repeat;
-    display: block;
-    height: 58px;
-    width: 152px;
-
-    span {
-      display: none;
+#logo-container{
+  width: 100%;
+  bottom: 40px;
+  position: fixed;
+  #lrg-mark {
+    img {
+      margin: 0;
+      padding: 0;
     }
 
-    @media screen and (min-width: 482px) {
-      left: 15px;
-      position: fixed;
+    @media screen and (min-width: 800px) {
+      display: block;
+      height: 40px;
+      width: 95%;
+      max-width: 1200px;
+      margin-left: auto;
+      margin-right: auto;
     }
-    @media screen and (max-width: 1280px) and (min-width: 482px){
+
+    @media screen and (max-width: 799px) {
       display: none;
     }
-    @media screen and (max-width: 481px) {
-      margin: 0 auto 20px auto;
-      background-position: center top;
-    }
-
+  }
 }
 
 header#banner {
-  text-transform: uppercase;
-  font-family: $block;
-  display: block;
+
+  width: 100%;
+  height: 221px;
   margin-left: auto;
   margin-right: auto;
-  background-color: $artsy_bg;
-  padding-bottom: 150px;
+  z-index: 999;
 
-  @media screen and (min-width: 482px) {
-    padding-top: 20px;
-    width: 900px;
-  }
-  @media screen and (max-width: 481px) {
+  @media screen and (max-width: 799px) {
     text-align: center;
     margin-bottom: -30px;
   }
 
+  &.stuck {
+    position:fixed;
+    top: -100px;
+    background-color: white;
+  }
+
   #header {
-    & a {
-      @include link-underline;
+    width: 95%;
+    max-width: 1200px;
+    margin-left: auto;
+    margin-right: auto;
+    padding-top: 40px;
+    position: relative;
+    height: 180px;
+    border-bottom: 1px solid $artsy-text;
+
+
+    .header-navigation{
+      margin-left: 250px;
+      height: 100px;
+      position: static;
+      ul{
+        float: left;
```

Comment:
@gristleism Floats can get rather un-weildy and these days there's often better alternatives. My go-to positioning tricks, in order of most used, are...

Lining things up in a row with inline-block
````css
.left, .center, .right {
  display: inline-block;
  vertical-align: top;
}
````
The caveat here is whitespace in the html can cause weird gaps. But html compression will strip all whitespace to alleviate this (Express.js does this by default).

Positioning something according to it's parent with position: relative & absolute

````css
.parent-menu {
  position: relative;
}
/* This will move to the top right of the parent-menu */
.child-hamburger-icon {
  position: absolute;
  top: 10px;
  right: 10px;
}
````

And finally [CSS3 flex box](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) which is more advanced so I recommend reading up on it to really understand but here's an example.

````css
.container {
  display: flex;
}
/* .left and .right will be 25% of the container width while .middle will be 50% 
.left {
  flex-grow: 1;
}
.middle {
  flex-grow: 2;
}
.right {
  flex-grow: 1;
}
````

Floats aren't evil by any means but they tend to not play that nice with other non-floating things and in my experience any use case for them can be solved by one of the less-unwieldy approaches above.

Food for thought, hope that was helpful!